packages/core/components/Layout/components/Sidebar/components/sidebar.styles.scss

```diff
@@ -0,0 +1,914 @@
+@import '@cdc/core/styles/v2/themes/_color-definitions.scss';
+
```

Comment:
can you write this is common CSS?

Reply:
I think we can later on but that's outside of the scope of this ticket.